### ISSUE TITLE
Crafting Pesto is actually useless

### ISSUE THREAD

**Issue by rubenpd** [CleverRaven/Cataclysm-DDA#26127]
**Describe the bug**  
Crafting "Pesto" as ingredient for "Spaghetti al Pesto" is a waste of ingredients and time, and Pesto has no other uses other than making spaghetti/pizza

**Expected behavior**  
Either it should be cheaper to make the pesto or the alternative of substituting it should be much higher

**Screenshots**  
https://cdn.discordapp.com/attachments/365907426270117888/498808523409326080/unknown.png

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version Jenkins 8025 and 8026
 - Graphics version Tiles
 - Mods loaded not relevant, pesto is vanilla

**Additional context**  
No additional context

**Comment by jeremyshannon** [comment]
Yeah, one portion of pesto for 5 Italian seasoning + everything else, vs just using the 5 Italian seasoning is pointless.  Salt is cheap, but cooking oil and pine nuts are uncommon.

 Maybe the pesto recipe should produce more than one portion? I think four portions would make it worthwhile, especially if you were low on Italian seasoning -- then it would be 1 cooking oil, 1 pine nuts, 5 Italian Seasoning, and 5 salt vs 20 Italian seasoning for the same four batches of Spaghetti/Pizza.

**Comment by rubenpd** [comment]
yes, and maybe make portions smaller, to fit the volume of the ingredients used as components

**Comment by KorGgenT** [comment]
i will address this in #27289
